staging/src/k8s.io/apiserver/pkg/cel/library/cost.go

```diff
@@ -152,6 +153,32 @@ func (l *CostEstimator) CallCost(function, overloadId string, args []ref.Val, re
 			cost := uint64(math.Ceil(float64(actualSize(args[0])) * common.StringTraversalCostFactor))
 			return &cost
 		}
+	case "validate":
```

Comment:
This switch is on `function`. I should it be changed to `overloadId` which is unique?

Reply:
overload ID would be more future proof.. It's not critically urgent since we don't have any overloads causing ambiguities at the moment..